evm/src/fuzz/strategies/state.rs

```diff
@@ -60,13 +60,20 @@ pub fn build_initial_state<DB: DatabaseRef>(db: &CacheDB<DB>) -> EvmFuzzState {
 
         // Insert basic account information
         state.insert(H256::from(*address).into());
-        state.insert(utils::u256_to_h256_le(info.balance).into());
-        state.insert(utils::u256_to_h256_le(U256::from(info.nonce)).into());
+        state.insert(utils::u256_to_h256_be(info.balance).into());
+        state.insert(utils::u256_to_h256_be(U256::from(info.nonce)).into());
```

Comment:
our unit tests capture this now right?

Reply:
yes, both `test_invariant_storage` and `test_fuzz_collection`